src/utils/util.js

```diff
@@ -88,15 +94,23 @@ function getServer(inputURL, teams) {
     return null;
   }
   let parsedServerUrl;
+  let secondoption = null;
   for (let i = 0; i < teams.length; i++) {
     parsedServerUrl = parseURL(teams[i].url);
 
     // check server and subpath matches (without subpath pathname is \ so it always matches)
-    if (parsedServerUrl.origin === parsedURL.origin && parsedURL.pathname.startsWith(parsedServerUrl.pathname)) {
+    if (testUrlsWithSubpath(parsedServerUrl, parsedURL)) {
       return {name: teams[i].name, url: parsedServerUrl, index: i};
     }
+    if (testUrlsIgnoringSubpath(parsedServerUrl, parsedURL)) {
+      // in case the user added something on the path that doesn't really belong to the server
+      // there might be more than one that matches, but we can't differentiate, so last one
+      // is as good as any other.
+      // e.g.: https://community.mattermost.com/core
+      secondoption = {name: teams[i].name, url: parsedServerUrl, index: i};
```

Comment:
Also, the capitalization here is incorrect
```suggestion
      secondOption = {name: teams[i].name, url: parsedServerUrl, index: i};
```

Reply:
yes, since we are not too strict on how people add servers, there might be a better match, for example two subpath servers on the same origin. so I just hold it and if there is nothing better just use that one.